point_angle_range_to_polygon.py

```diff
@@ -12,47 +12,67 @@ def createPointAngleRangePolygons(points, outputPolygons, azimuthField, rangeFie
     azimuthField = azimuthField
     rangeField = rangeField
     beamWidthField = beamWidthField
-    #Polygon feature class will created by the tool. 
     outputPolygonFeatures = outputPolygons
     polygons = []
-    
-    cursorFields = ["SHAPE@", azimuthField, rangeField, beamWidthField]
+    #Setup field list for point search cursor and polygon insert cursor    
+    cursorFields = [f.name for f in arcpy.ListFields(pointFeatures)]
+    cursorFields.remove('SHAPE')
+    cursorFields.append("SHAPE@")
+    polygonInsCursor = arcpy.da.InsertCursor(outputPolygonFeatures, cursorFields)
     with arcpy.da.SearchCursor(pointFeatures, cursorFields) as cursor:
         for row in cursor:
-            centerPoint = row[0]
-            azimuth = row[1]
-            rangeDist = row[2]
-            beamWidth = row[3]
+            centerPoint = row[len(cursorFields) - 1]
```

Comment:
It would be the index of SHAPE@ and think using it is better here. 
I made a habit of not trusting ArcGIS generated field names like OBJECTID vs FID in feature classes and shapefiles. The da cursors seem handle the special field tokens the same way in all data types.